pom.xml

```diff
@@ -142,6 +142,7 @@ under the License.
 		<zookeeper.version>3.5.9</zookeeper.version>
 		<curator.version>5.2.0</curator.version>
 		<avro.version>1.11.1</avro.version>
+		<jackson-bom.version>2.13.4</jackson-bom.version>
```

Comment:
I actually thought of putting it right below `<flink.shaded.jackson.version/>` further up in [line 126](https://github.com/apache/flink/blob/0025fab2c7bd4ddfd0f4788143204d5d24aa2686/pom.xml#L126). But I noticed that it would screw up the lexicographical order. So, leaving it like that is good enough, I guess. Could we add some comment here explaining why we have a separate jackson dependency here along the shaded version? Something along the lines of
> Version for transitive Jackson dependencies that are not used within Flink itself.

Reply:
Introducing the property could be done in a separate hotfix commit.

Reply:
done in a separate commit